about.html

```diff
@@ -0,0 +1,134 @@
+            />
+            <ul class="about-menu">
+              <li class="close"><a href="./about.html">About</a></li>
+              <li class="close"><a href="#events">Events</a></li>
+              <li class="close"><a href="#">Join</a></li>
+              <li class="close"><a href="#">Sponsor</a></li>
+              <li class="close"><a href="#">News</a></li>
+              <li class="book-now close"><a href="#"> Book Now</a></li>
+            </ul>
+          </div>
+        </div>
+      </div>
+
+      <main class="about-main">
+        <div class="about-title">
+          <h2>"South Africa's best marketplace for live events"</h2>
+          <h1>LIVE CONCERTS MUSIC AT THE LAKE 2022</h1>
+          <p class="about-para1">
+            This is the event never to be missed, enjoy your favourite music
+            with a lovely view while watching a beautiful sunset Lorem ipsum,
+            dolor sit amet consectetur adipisicing elit. Natus, doloremque enim!
+            Necessitatibus fuga voluptas voluptatibus. Quia, laboriosam deleniti
+            omnis nam aliquam libero accusantium dolorum dolor corrupti placeat
+            tempore, nesciunt debitis.
+          </p>
+          <p class="about-para2">
+            For further details about music at the lake 2022, please contact us
+            by email
+          </p>
+          <p class="para2-email">musicatthelake@liveconcerts.co.za</p>
+        </div>
+      </main>
+    </section>
+    <section class="LC-logo">
+      <h2>Live Concerts Music At The Lake Logo 2022</h2>
+      <p>The logo for this event was made by Siphosethu Mahlangu</p>
+      <hr class="breakline" />
+      <img src="./images/white-logo1.jpg" alt="live-concerts-logo" />
+    </section>
+    <section class="about-events">
+      <div class="past-events">
+        <h2>See our past Live Events</h2>
+        <hr class="breakline" />
+        <p>
+          Our last two live events which took place in Durban and Cape Town
+          South Africa
+        </p>
+        <img
+          class="img1"
+          src="./images/event-y2.jpg"
+          alt="Picture of singing group"
+        />
+        <img
+          class="img2"
+          src="./images/past-events2.jpg"
+          alt="Picture of live concert"
+        />
+      </div>
+      <div class="partners">
+        <h2>Partners</h2>
+        <hr class="breakline" />
+        <div class="partners-img">
+          <div>
+            <img src="./images/mozilla.png" alt="Mozilla brand" />
+          </div>
+          <div>
+            <img src="./images/google.png" alt="Google brand" />
+          </div>
+          <div>
+            <img src="./images/naver.png" alt="Naver brand" />
+          </div>
+          <div>
+            <img src="./images/kakao.png" alt="Kakao brand" />
+          </div>
+          <div>
+            <img src="./images/airbnb.png" alt="Airbnb brand" />
+          </div>
+        </div>
+      </div>
+      <div class="about-footer">
```

Comment:
- [ ] It would be semantic if you used a `<footer></footer>` tag for the footer instead of a div 👍